stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/attachments/content/AudioRecordAttachmentContent.kt

```diff
@@ -16,79 +16,165 @@
 import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxSize
 import androidx.compose.foundation.layout.fillMaxWidth
 import androidx.compose.foundation.layout.height
 import androidx.compose.foundation.layout.padding
 import androidx.compose.foundation.layout.width
 import androidx.compose.foundation.shape.CircleShape
 import androidx.compose.material.Card
-import androidx.compose.material.Icon
+import androidx.compose.material.ExperimentalMaterialApi
 import androidx.compose.material.IconButton
-import androidx.compose.material.Slider
 import androidx.compose.material.Surface
 import androidx.compose.material.Text
-import androidx.compose.material.TextButton
+import androidx.compose.material3.Icon
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
 import androidx.compose.runtime.getValue
-import androidx.compose.runtime.mutableFloatStateOf
-import androidx.compose.runtime.mutableStateOf
-import androidx.compose.runtime.remember
-import androidx.compose.runtime.setValue
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.graphics.Color
-import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.style.TextAlign
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
-import io.getstream.chat.android.client.ChatClient
-import io.getstream.chat.android.client.audio.AudioPlayer
-import io.getstream.chat.android.client.audio.AudioState
+import androidx.lifecycle.viewmodel.compose.viewModel
 import io.getstream.chat.android.client.extensions.duration
-import io.getstream.chat.android.compose.R
+import io.getstream.chat.android.client.extensions.waveformData
+import io.getstream.chat.android.client.utils.attachment.isAudioRecording
+import io.getstream.chat.android.client.utils.message.isMine
+import io.getstream.chat.android.compose.state.messages.attachments.AttachmentState
+import io.getstream.chat.android.compose.ui.components.audio.WaveformSlider
 import io.getstream.chat.android.compose.ui.theme.ChatPreviewTheme
 import io.getstream.chat.android.compose.ui.theme.ChatTheme
+import io.getstream.chat.android.compose.ui.util.padding
+import io.getstream.chat.android.compose.ui.util.size
+import io.getstream.chat.android.compose.viewmodel.messages.AudioPlayerViewModel
+import io.getstream.chat.android.compose.viewmodel.messages.AudioPlayerViewModelFactory
+import io.getstream.chat.android.extensions.isInt
 import io.getstream.chat.android.models.Attachment
+import io.getstream.chat.android.ui.common.state.messages.list.AudioPlayerState
 import io.getstream.chat.android.ui.common.utils.DurationFormatter
 
-/**
- * Represents fallback content for unsupported attachments.
- *
- * @param modifier Modifier for styling.
- */
+@Deprecated(
+    message = "Use AudioRecordAttachmentContent instead",
+    replaceWith = ReplaceWith("AudioRecordAttachmentContent"),
+)
```

Comment:
It would be good if we can add the `imports` argument for the proper replacement.

```kotlin
imports = ["io.getstream.chat.android.compose.ui.attachments.content"],
```